bitnami/redis-cluster/templates/svc-cluster-external-access.yaml

```diff
@@ -5,6 +5,7 @@
 
 {{- range $i, $e := until $nodesCount }}
 {{- $targetPod := printf "%s-%d" (printf "%s" $fullName) $i }}
+{{- $_ := set $ "instance" $i }}
```

Comment:
I think this is not needed. Maybe you can implement something similar to what is done with `(set $ "dataNodeLoopId" $i)` [here](https://github.com/bitnami/charts/blob/master/bitnami/mongodb-sharded/templates/shard/shard-data-statefulset.yaml#L42-L44).